src/ui/SortDropdown/SortDropdown.ts

```diff
@@ -65,8 +65,19 @@ export class SortDropdown extends Component {
     this.buildDropdown();
   }
 
+  private deletePreviousDropdown() {
```

Comment:
I think `clearDropdown` would be a better name.
It's not deleting the "previous" one, it's emptying the current dropdown.